openshift/system/Dockerfile

```diff
@@ -70,7 +70,6 @@ RUN source /opt/app-root/etc/scl_enable \
 RUN chgrp root /opt/system/
 
 ADD . ./
-ADD config/docker/* ./config/
```

Comment:
we also need to change porta-midstream with this

Reply:
Yep!